creusot/tests/should_succeed/iterators/08_collect_extend.mlcfg

```diff
@@ -747,11 +749,13 @@ module C08CollectExtend_Extend
     goto BB4
   }
   BB4 {
+    assume { Resolve1.resolve iter_old_9 };
```

Comment:
This is extraneous and caused by not calling `emit_resolve` at the correct place. That function should elide trivial resolves like for `Ghost<T>`.